### PR TITLE
KRACOEUS-8129 fix fop error in: CommitteeBatchCorrespondenceServiceTest

### PR DESCRIPTION
**Pull request by boneypolus**
Committed the files.

### CODE REVIEW

**Comment by kualiCoeusCi** [comment]
Can one of the admins verify this patch?

File: coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
     
     public static final String BROWSER_PROTOCOL = "http";
     public static final String BROWSER_ADDRESS = "127.0.0.1";
-    public static final String PORTAL_ADDRESS = "kc-dev";
+    public static final String PORTAL_ADDRESS = "";
```
Comment (blackcathacker) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: Why did you change this? That doesn't seem correct.

Comment (boneypolus) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: When PORTAL_ADDRESS = "kc-dev", we can't access the image using the relative path. Its always need kc-dev at first of the path. Since the template is using across the application we can't change the path to kc-dev. So we removed.

Comment (blackcathacker) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: And all other integration tests still pass?

**Comment by blackcathacker** [comment]
test this please

File: coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java (hunk 1 above)
Comment (blackcathacker) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: @leoherbie can you look at this change? This is weird to me. Why do we even have this value if it isn't necessary?

Comment (leoherbie) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: I agree.  This doesn't seem right.  Why exactly don't the relative paths work?  I think this will break our other int tests

Comment (boneypolus) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: We tried other int tests after this change and its working fine.

Comment (leoherbie) [inline coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java]: did you run the full suite.  It doesn't take that long.  I'd suggest doing that.``` mvn clean install -Pitests```

**Comment by rmancher** [comment]
@leoherbie thoughts on this?
@boneypolus remove PORTAL_ADDRESS if not required and run full suite to see if all tests are fine

**Comment by blackcathacker** [comment]
I for one would still love to know why this change was required. Can you supply a diff of the changes you made to the stylesheet? And describe why the url matters to the fop and the stylesheet?

**Comment by boneypolus** [comment]
Yes, we did changes in template. Corresponding table name is 'PROTO_CORRESP_TEMPL' , in this table we edited two rows of data to fix the error mentioned in jira.

I. Row with PROTO_CORRESP_TYPE_CODE = 20(DEFAULT-20-RenewalReminderLetter#1.xsl) , we did two changes,

1: To solve,
 ERROR org.apache.fop.apps.FOUserAgent - Image not found. URI: /export/home/www/https/tomcat5.0.25/webapps/coeus/images/couhes_byline2.gif. (No context info available) ,
we changed the url to static/images/couhes_byline2.gif and added a dummy image with the same name. Also removed PORTAL_ADDRESS ="kc-dev" from KcIntegrationTestBaseLifecycle.java

2: To solve, 
ERROR org.apache.fop.apps.FOUserAgent - Invalid property value encountered in column-width="proportional-column-width(9)": org.apache.fop.fo.expr.PropertyException: proportional-column-width() function may only be used when fo:table has table-layout="fixed".; property:'column-width' (No context info available) 
org.apache.fop.fo.expr.PropertyException: proportional-column-width() function may only be used when fo:table has table-layout="fixed".; property:'column-width'  ,
we added table-layout="fixed" in table definition.

II. Row with PROTO_CORRESP_TYPE_CODE = 23 (DEFAULT-23-ReminderToIrbNotification#1.xsl), we did one change to solve the same 1st fop error (image not found).

**Comment by blackcathacker** [comment]
So this still makes me nervous. There must be something about the current directory when running jetty from these it-tests. There must be a better way to handle that than having the it-test jetty server startup with / being KC instead of using /kc-dev. I mean the change to remove kc-dev isn't serious, but at the same time its a hack to get this one test to work when we should understand why FOP is looking in the wrong place when running it-tests.

**Comment by leoherbie** [comment]
I agree with @blackcathacker. Could you explain why this isn't an issue when the app is deployed under the context kc-dev which is essentially what is happening here.

**Comment by rmancher** [comment]
@boneypolus, I see that you modified the template to use external image url as parameter to get the image path.
We are retaining portal address in this case - right?

**Comment by boneypolus** [comment]
@rmancher , Yes we are retaining portal address as it is and tried all the scenarios and its working fine.

**Comment by blackcathacker** [comment]
Much more understandable solution. Thank you.

**Comment by leoherbie** [comment]
thanks for this.  it is a much better solution.
